cli/admin/src/main/java/io/pravega/cli/admin/segmentstore/tableSegment/SetKeySerializerCommand.java

```diff
@@ -0,0 +1,59 @@
+/**
+ * Copyright Pravega Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.pravega.cli.admin.segmentstore.tableSegment;
+
+import com.google.common.collect.ImmutableMap;
+import io.pravega.cli.admin.CommandArgs;
+import io.pravega.cli.admin.serializers.AbstractSerializer;
+import io.pravega.cli.admin.serializers.ContainerKeySerializer;
+import io.pravega.cli.admin.serializers.SltsKeySerializer;
+
+import java.util.Map;
+
+public class SetKeySerializerCommand extends TableSegmentCommand {
```

Comment:
Should we collapse these two commands SetKeySerializerCommand and SetValueSerializerCommand into a single SetSerializer command that set serializer for both key and value in a single command?
Just a thought...

Reply:
It is a good idea but I believe as we add more serializers and even allow users to make their own serializers. This can be more flexible to the user.
@andreipaduroiu what are your thoughts?

Reply:
I've generalised this into a single command `set-serializer`.